components/dashboard/src/App.tsx

```diff
@@ -193,223 +69,21 @@ function App() {
-                            single machine without requiring a Kubernetes cluster.{" "}
-                            <a
-                                className="gp-link hover:text-gray-600"
-                                href="https://www.gitpod.io/community-license?utm_source=local-preview"
-                            >
-                                Request a community license
-                            </a>{" "}
-                            or{" "}
-                            <a
-                                className="gp-link hover:text-gray-600"
-                                href="https://www.gitpod.io/contact/sales?utm_source=local-preview"
-                            >
-                                contact sales
-                            </a>{" "}
-                            to get a professional license for running Gitpod in production.
-                        </Alert>
-                    </div>
-                )}
-                <AppNotifications />
-                <Switch>
-                    <Route path={projectsPathNew} exact component={NewProject} />
-                    <Route path="/open" exact component={Open} />
-                    <Route path="/setup" exact component={Setup} />
-                    <Route path={workspacesPathMain} exact component={Workspaces} />
-                    <Route path={settingsPathAccount} exact component={Account} />
-                    <Route path={usagePathMain} exact component={Usage} />
-                    <Route path={settingsPathIntegrations} exact component={Integrations} />
-                    <Route path={settingsPathNotifications} exact component={Notifications} />
-                    <Route path={settingsPathBilling} exact component={Billing} />
-                    <Route path={settingsPathPlans} exact component={Plans} />
-                    <Route path={settingsPathVariables} exact component={EnvironmentVariables} />
-                    <Route path={settingsPathSSHKeys} exact component={SSHKeys} />
-                    <Route path={settingsPathPersonalAccessTokens} exact component={PersonalAccessTokens} />
-                    <Route
-                        path={settingsPathPersonalAccessTokenCreate}
-                        exact
-                        component={PersonalAccessTokenCreateView}
-                    />
-                    <Route
-                        path={settingsPathPersonalAccessTokenEdit + "/:tokenId"}
-                        exact
-                        component={PersonalAccessTokenCreateView}
-                    />
-                    <Route path={settingsPathPreferences} exact component={Preferences} />
-                    <Route path={projectsPathInstallGitHubApp} exact component={InstallGitHubApp} />
-                    <Route path="/from-referrer" exact component={FromReferrer} />
-
-                    <AdminRoute path="/admin/users" component={UserSearch} />
```

Comment:
@selfcontained These seem to have gone missing if I'm not mistaken...?

Reply:
These should all [be here](https://github.com/gitpod-io/gitpod/pull/15396/files/518cc2187e2f7429f2e03f602141c15b94172068#diff-7d06461d6c55d4b24d38b66745a49c9a28e9992fccb8eb7459bc91f4d4583e11R171-R310)  in `<AppRoutes/>`